cl_hubeau/water_services/water_services_scraper.py

```diff
@@ -0,0 +1,240 @@
+# -*- coding: utf-8 -*-
+"""
+low level class to collect data from the services indiators API from hub'eau
+"""
+import pandas as pd
+
+from cl_hubeau.session import BaseHubeauSession
+from cl_hubeau.exceptions import UnexpectedArguments
+
+
+class WaterServicesSession(BaseHubeauSession):
+    """
+    Base session class to handle the services WaterServices API
+    """
+
+    DOC_URL = "https://hubeau.eaufrance.fr/page/api-indicateurs-services"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(version="0.3.1", *args, **kwargs)
+
+        # Set default size for API queries, based on hub'eau indicator's doc
+        self.size = 5000
+
+    def get_communes(self, **kwargs):
+        """
+        Lister les services par commune
+        Endpoint /v0/indicateurs_services/communes
+
+        Appel des indicateurs de performance des services publics d'eau et
+        d'assainissement par commune (un minimum, maximum et moyenne
+        sont calculés quand il existe plusieurs services publics
+        sur une même commune)
+
+        Source de données : Observatoire des services publics de l'eau
+        et de l'assainissement : prix de l'eau et performance des services
+        Site web : http://services.eaufrance.fr/
+
+        Si la valeur du paramètre size n'est pas renseignée, la taille
+        de page par défaut : 2000, taille max de la page : 10000.
+        La profondeur d'accès aux résultats est : 20000, calcul de
+        la profondeur = numéro de la page * nombre maximum de
+        résultats dans une page.
+
+        Doc: https://hubeau.eaufrance.fr/page/api-indicateurs-services
+        """
+
+        params = {}
+
+        try:
+            params["annee"] = kwargs.pop("annee")
+        except KeyError:
+            pass
```

Comment:
You could maybe use `kwargs.pop("annee", None)` instead to avoid the need for repeated error handling.